caffe2/CMakeLists.txt

```diff
@@ -1191,15 +1191,23 @@ if(BUILD_MOBILE_BENCHMARK)
   endforeach()
 endif()
 
-if(BUILD_MOBILE_TEST)
+  if(BUILD_MOBILE_TEST)
```

Comment:
Mobile tests don't seem to be the good place for these tests. Consider putting them into the `BUILD_TEST` section.

Reply:
I moved them into BUILD_TEST